library/redis

```diff
@@ -4,6 +4,16 @@ Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
 GitRepo: https://github.com/docker-library/redis.git
 
+Tags: 7.0-rc1, 7.0, 7.0-rc, 7.0-rc1-bullseye, 7.0-bullseye, 7.0-rc-bullseye
```

Comment:
Do we want to avoid plain tags like `7.0` on an `rc` release?

Reply:
Ah yeah, I guess so -- good catch.

Reply:
Updated via https://github.com/docker-library/redis/commit/5800387c04b3a07957e578258e0438187dd9e5d2 (updated commit here coming in just a second).